packages/libdb.rb

```diff
@@ -27,10 +27,14 @@ def self.patch
   def self.build
     Dir.chdir 'build_unix' do
       system "../dist/configure #{CREW_CONFIGURE_OPTIONS} \
+      --disable-java \
+      --disable-sql \
+      --disable-stl \
+      --disable-tcl \
```

Comment:
Why so many disables?  Can we enable at least some of these with other dependencies?

Reply:
This is the libdb that Oracle took over and closed off over a decade ago.

It really only exists for compatiblity reasons. 

The disables are as per the debian build options to reduce the attack surface.

Reply:
Also, this requires gcc 14 to build...